packages/angular-table/src/index.ts

```diff
@@ -1,96 +1,61 @@
-import {
-  computed,
-  effect,
-  inject,
-  Injector,
-  runInInjectionContext,
-  type Signal,
-  signal,
-  untracked,
-} from '@angular/core'
+import { computed, effect, inject, Injector, signal } from '@angular/core'
 import {
   createTable,
   RowData,
-  type Table,
   TableOptions,
   TableOptionsResolved,
+  TableState,
+  type Table,
 } from '@tanstack/table-core'
-import { proxifyTable } from './proxy'
 import { lazyInit } from './lazy-signal-initializer'
 
 export * from '@tanstack/table-core'
 
-export {
-  FlexRenderDirective,
-  FlexRenderComponent,
-  injectFlexRenderContext,
-} from './flex-render'
+export { FlexRenderDirective } from './flex-render'
```

Comment:
This should be reverted, both `FlexRenderComponent` and `injectFlexRenderContext` are needed to be able to render components via columns definition